code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm

```diff
@@ -174,7 +174,7 @@
 	vines += newVine
 	if(isliving(the_target))
 		var/mob/living/L = the_target
-		L.Paralyze(20)
+		L.Knockdown(2 SECONDS) //Skyrat EDIT - Removes hardstun, bye!
```

Comment:
This is fine because it makes people drop weapons.

Reply:
Paralyse prevents picking up weapons, and 20 ticks on average lag ~4 seconds of a hardstun where you cannot do anything. This also worked on simplemobs, versus knockdown which does not.